modules/core/petra/petra-concurrent/src/test/java/com/liferay/petra/concurrent/ConcurrentMapperHashMapTest.java

```diff
@@ -943,7 +1089,7 @@ private void _assertEventQueue(Event... expectedEvents) {
 		new ConcurrentTypeReferenceHashMap();
 	private final Key _testKey = new Key("testKey");
 	private final Value _testValue = new Value("testValue");
-	private final Value _testValue2 = new Value("testValue2");
+	private final Value _testValueAnother = new Value("testValueAnother");
```

Comment:
@tinatian why this renaming?

Reply:
@shuyangzhou Just think it is weird since there is no testValue1.